tests/acceptance/features/webUISharingInternalUsersToRoot/shareWithUsers.feature

```diff
@@ -96,7 +103,9 @@ Feature: Sharing files and folders with internal users
 
 
   Scenario: share a folder with another internal user who unshares the folder
-    Given user "Brian" has logged in using the webUI
+    Given user "Brian" has created folder "simple-folder"
+    And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
+    And user "Brian" has logged in using the webUI
     When the user renames folder "simple-folder" to "new-simple-folder" using the webUI
```

Comment:
let's use `new-simple-folder` directly without renaming, specially that renaming sounds like its not part of the test